src/native/external/cgmanifest.json

```diff
@@ -55,8 +55,8 @@
       "Component": {
         "Type": "git",
         "Git": {
-          "RepositoryUrl": "https://github.com/jtkukunas/zlib",
-          "CommitHash": "bf55d56b068467329f5a6f29bee31bc80d694023"
+          "RepositoryUrl": "https://github.com/intel/zlib",
+          "CommitHash": "6160a8f20c3626aec2f8b0fda5bf2e65bfe31781"
```

Comment:
Glad to see the intel zlib is now under the intel org.

I can see that 6160a is the commit for the 1.2.13 release: https://github.com/intel/zlib/tags